src/backend/tests/models/charge_types/test_person_felony.py

```diff
@@ -0,0 +1,193 @@
+    "1643772C",  # Computer Crime—Theft of an Intimate Image;
+    # "164395",  # Robbery III;
+    # "164405",  # Robbery II;
+    "164415",  # Robbery I;
+    "1648863",  # Tree Spiking (Injury);
+    "166070",  # Aggravated Harassment;
+    "166087",  # Abuse of Corpse I;
+    # "166165",  # Bias Crime I;
+    # "166220",  # Unlawful Use of a Weapon;
+    "166275",  # Inmate In Possession of Weapon;
+    "1663853",  # Felony Possession of a Hoax Destructive Device;
+    "166643",  # Unlawful Possession of Soft Body Armor as defined in Crime Category 6 (OAR 213-018-0090(1));
+    "167012",  # Promoting Prostitution;
+    "167017",  # Compelling Prostitution;
+    "167057",  # Luring a Minor;
+    "1673204",  # Felony Animal Abuse I;
+    "167322",  # Aggravated Animal Abuse I;
+    "468951",  # Environmental Endangerment;
+    "4757526A",  # Manufacturing or Delivering a Schedule IV Controlled Substance Thereby Causing Death to a Person;
+    "475908",  # Causing Another to Ingest a Controlled Substance as defined in Crime Categories 8 and 9 (OAR 213-019-0007 and 0008);
+    "475910",  # Unlawful Administration of a Controlled Substance as defined in Crime Categories 5, 8, and 9 (OAR 213-019-0007, -0008, and -0011);
+    # "475B359",  # Arson Incident to Manufacture of Cannabinoid Extract I;
+    # "475B367",  # Causing Another Person to Ingest Marijuana;
+    # "475B371",  # Administration to Another Person Under 18 Years of Age;
+    "6099903B",  # Maintaining Dangerous Dog;
+    # "811705",  # Hit and Run Vehicle (Injury);
+    # "8130105",  # Felony Driving Under the Influence of Intoxicants (as provided in OAR 213-004-0009);
+    "8304752",  # Hit and Run Boat;
+    "8373652B",  # Unlawful Operation of Weaponized Unmanned Aircraft System;
+    "8373652C",  # Unlawful Operation of Weaponized Unmanned Aircraft System;
+]
+
+person_felony_statutes_with_subsection = [
+    "1631603",  # [contains subsection] # Felony Assault; class C felony
+    "1631874",  # [contains subsection] # Felony Strangulation; class C felony
+    "1643772C",  # [contains subsection] # Computer Crime—Theft of an Intimate Image; class C misdemeanor
+    "1648863",  # [contains subsection] # Tree Spiking (Injury); class B felony
+    "1663853",  # [contains subsection] # Felony Possession of a Hoax Destructive Device; class C felony
+    "1673204",  # [contains subsection] # Felony Animal Abuse I; class C felony
+    "4757526A",  # [contains subsection] # Manufacturing or Delivering a Schedule IV Controlled Substance Thereby Causing Death to a Person; adjacent to a class B felony
+    "6099903B",  # [contains subsection] # Maintaining Dangerous Dog; class C felony
+    "8304752",  # [contains subsection] # Hit and Run Boat; no level
+    "8373652B",  # [contains subsection] # Unlawful Operation of Weaponized Unmanned Aircraft System; class C felony
+    "8373652C",  # [contains subsection] # Unlawful Operation of Weaponized Unmanned Aircraft System; class B felony
+]
```

Comment:
This list seems to be duplicated: https://github.com/codeforpdx/recordexpungPDX/pull/870/files#diff-69da7724337f6dd0eaca7583209e1ba2R21

Reply:
That's intentional, so that if we update the charge type logic without updating the tests we will know it because of the failing test.

Reply:
Was this list independently written though? I don't see the point if all future changes will just involve modifying the original list and then copy pasting those changes to replace this duplicated list.

Reply:
belt and suspenders?

Reply:
I see what you mean though